app/models/storage.rb

```diff
@@ -872,14 +879,12 @@ def vm_scan_affinity
   end
 
   def self.batch_operation_supported?(operation, ids)
```

Comment:
Have you checked the callers of this? What operations can be passed in? Have they all been refactored to `support_` ?